<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFormUi</class>
 <widget class="QMainWindow" name="MainFormUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ЛР 11-13</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="resultsTable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>Обмен</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Выбор</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Включение</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Шелла</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Быстрая</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Встроенная</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Подсчетом</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Включить</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Сравнения</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Перестановки</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Время (нс)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Массив отсортирован</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
       <property name="flags">
        <set>ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="0" column="4">
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
       <property name="flags">
        <set>ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="1" column="3">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="1" column="4">
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
       <property name="flags">
        <set>ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="2" column="3">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="2" column="4">
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
       <property name="flags">
        <set>ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="3" column="2">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="3" column="3">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="3" column="4">
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
       <property name="flags">
        <set>ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="4" column="2">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="4" column="3">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="4" column="4">
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
       <property name="flags">
        <set>ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="5" column="1">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="5" column="2">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="5" column="3">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="5" column="4">
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="6" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
       <property name="flags">
        <set>ItemIsEditable|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item row="6" column="1">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="6" column="2">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="6" column="3">
       <property name="text">
        <string>0</string>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
      <item row="6" column="4">
       <property name="text">
        <string/>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <property name="flags">
        <set>ItemIsEnabled</set>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="sortBtn">
        <property name="text">
         <string>Сортировать</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Размер массива:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="arrSize">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>25</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Размер (кб):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="arrKbytes">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="exitBtn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="exit">
          <normaloff>../../../../../../../.designer/backup</normaloff>../../../../../../../.designer/backup</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
